rat/variables.py

```diff
@@ -190,14 +190,15 @@ def size(self):
             return len(self.subscript.base_df.index)
 
     def code(self):
-        return f"assigned_param__{self.ops_param.name}"
+        return f"{self.ops_param.name}"
 
 
 @dataclass
 class SubscriptUse:
     names: Tuple[str]
     df: pandas.DataFrame
     subscript: Subscript
+    unique_id: str
```

Comment:
What is this attribute used for?

Reply:
In a program you can have different subscript uses with the same shifts and names that correspond to differerent dataframes.

So like:
```
y ~ normal(mu[group], sigma[group]);
mu[group] ~ normal(somethingelse[group], ...);
...
```

I'm now saving subscript uses in a global data structure, so I need to make sure they have their own name for generated code.

I thought about putting this unique_id generation as part of the SubscriptUse class so we didn't need to pass a unique_id in, but decided to use leave it in the compiler code itself.

Reply:
Oh okay that makes sense 👍